tsuru/client/apps.go

```diff
@@ -210,70 +214,50 @@ func (c *AppCreate) Run(context *cmd.Context, client *cmd.Client) error {
 }
 
 type AppUpdate struct {
-	description string
-	platform    string
-	plan        string
-	pool        string
-	teamOwner   string
-	imageReset  bool
-	noRestart   bool
-	tags        cmd.StringSliceFlag
-	fs          *gnuflag.FlagSet
+	args tsuru.UpdateApp
+	fs   *gnuflag.FlagSet
 	cmd.GuessingCommand
 	cmd.ConfirmationCommand
+
+	memory, cpu string
 }
 
 func (c *AppUpdate) Info() *cmd.Info {
 	return &cmd.Info{
 		Name:  "app-update",
-		Usage: "app-update [-a/--app appname] [--description/-d description] [--plan/-p plan name] [--pool/-o pool] [--team-owner/-t team owner] [--platform/-l platform] [-i/--image-reset] [--tag/-g tag]...",
-		Desc: `Updates an app, changing its description, tags, plan or pool information.
-
-The [[--description]] parameter sets a description for your app.
-
-The [[--no-restart]] parameter sets the specified parameters to be updated without restarting the application
-
-The [[--plan]] parameter changes the plan of your app.
-
-The [[--pool]] parameter changes the pool of your app.
-
-The [[--team-owner]] parameter sets owner team for an application.
-
-The [[--platform]] parameter sets a platform for an application.
-
-The [[--image-reset]] parameter rebuilds the platform of an application.
-
-The [[--tag]] parameter sets a tag for your app. You can set
-multiple [[--tag]] parameters.`,
+		Usage: "app-update [-a/--app appname] [--description/-d description] [--plan/-p plan name] [--pool/-o pool] [--team-owner/-t team owner] [--platform/-l platform] [-i/--image-reset] [--cpu cpu] [--memory memory] [--tag/-g tag]...",
```

Comment:
@cezarsa I miss whats the format of --memory, "500mb", "500M" or "500Mib" ?
also for `--cpu` param.

Reply:
We're using kubernetes resource lib to parse this value, I'll add a description including the supported suffixes.

Reply:
Done